src/ActorMultiVertex.cpp

```diff
@@ -381,7 +372,8 @@ void ActorMultiVertex::SetVertsFromSplinesInternal(size_t num_splines, size_t of
 void ActorMultiVertex::SetVertsFromSplines()
 {
 	if(AMV_DestTweenState().vertices.empty()) { return; }
-	switch(AMV_DestTweenState()._DrawMode)
+	const auto drawMode = AMV_DestTweenState()._DrawMode; // cache this value to avoid repeated calls when switching
```

Comment:
Where are the repeated calls? This seems unnecessary.

Reply:
I thought a switch expression evaluated independently against each case in C++, but this probably isn't a big deal.